projects/aca-content/src/lib/services/content-management.service.ts

```diff
@@ -85,7 +85,7 @@ interface RestoredNode {
   providedIn: 'root'
 })
 export class ContentManagementService {
-  private readonly createMenuButtonSelector = 'app-create-menu button';
+  private readonly createMenuButtonSelector = 'app-toolbar-menu button[id="app.toolbar.create"]';
```

Comment:
@AleksanderSklorz, I tried using the '#' approach before as well but it was not abled to pickup the element and threw an error

Reply:
can you paste here how that selector was looking when you tried?

Reply:
I think I understand why you had a problem with that - because there are dots in id... So I guess you have two choices: 
[id="app.toolbar.create"] or #app\.toolbar\.create (with slashes). I think you may have it for now